src/reports/components/report-sections/tool-link.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NewTabLink } from 'common/components/new-tab-link';
+import { NamedFC } from 'common/react/named-fc';
+import { toolName } from 'content/strings/application';
+
+export const ToolLink = NamedFC('ToolLink', () => (
+    <NewTabLink
```

Comment:
will this be shown for a11y insights service report as well? will the text be :

Get more information and download Accessibility Insights Service?
(we can't download the service)

Reply:
never mind. tool name will always be accessibility insights web.